RocketControlUnitGUI/src/routes/data/+page.svelte

```diff
@@ -1,11 +1,39 @@
-
 <script lang="ts">
+	import LineChart from './LineChart.svelte'; 
+	import MapChart from './MapChart.svelte';
+	import RocketChart from './RocketChart.svelte';
+	import {Canvas} from '@threlte/core';
+  </script>
+  
+  <main>
+	<!-- <div style="display: flex; flex-direction: column; align-items: center;">
+	  <LineChart collection="Rcupressure" />
+	  <LineChart collection="RcuTemperature" />
+	 <LineChart collection="Imu" />
+	</div> -->
+<!-- 	
+	<MapChart />  -->
```

Comment:
should this be commented out?

Reply:
yeah, need to tweak that page cause I need to use divs for css, so that everything looks in its proper place